### ISSUE TITLE
New subgraph series to supersede the outdated names

### ISSUE THREAD

**Issue by PaulRBerg** [sablier-labs/subgraphs#57]
### As discussed in https://github.com/sablier-labs/subgraphs/discussions/53

The task is to create new subgraph series, as it is not possible to rename subgraphs on The Graph. The existing subgraphs will continue working.

- [ ] New series for Flow to remove `v2`
- [ ] New series for Lockup to remove `v2`
- [ ] New series for the Airdrop contracts to remove `v2` and `ms` (acronym for Merkle Streamer)

It'd be great if we could do this before/ during the Lockup v2.0.0 launch in Q1 2025 — I imagine that @gavriliumircea will have to go through the slog of updating all subgraphs anyway.

P.S. we should also create a PR in [DeFiLlama](https://github.com/DefiLlama/DefiLlama-Adapters/blob/6b88947f7c69d9baa21ffedfeebf30cfab305baf/projects/sablier-v2/index.js) to update the endpoints. **edit**: this is [done](https://github.com/DefiLlama/DefiLlama-Adapters/pull/12815) now

**Comment by PaulRBerg** [comment]
@razgraf @gavriliumircea will you be able to include this task before the release next week?

The longer we let it like this, the bigger the refactor it will be in the future. And the `v2` name in the subgraph will confuse developers/integrators.

I think now it's a great time given that it was a major release.

**Comment by gavriliumircea** [comment]
Just so we are clear should we maintain the previous versions and the new ones?

**Comment by PaulRBerg** [comment]
> Just so we are clear should we maintain the previous versions and the new ones?

Yes, to not cause any downtime. But the previous versions should not receive any further updates (ever).

> Also for Lockups if we simply remove the V2 we will have naming conflicts with some of the V1 subgraphs

Why? We can name them like so: `sablier-lockup`.

**Comment by PaulRBerg** [comment]
@razgraf you can update the OP to add the todo list

**Comment by PaulRBerg** [comment]
🎉
